site/src/routes/index.svelte

```diff
@@ -318,11 +319,11 @@ npm run dev & open http://localhost:5000
 
 	<p class="linkify">Svelte is free and open source software, made possible by the work of dozens of volunteers. <a href="https://github.com/sveltejs/svelte">Join us!</a></p>
 
-	{#each contributors as contributor}
+	{#each contributors as contributor, i}
 		<a
 			class="contributor"
-			style="background-image: url({contributor.src})"
-			href="https://github.com/{contributor.name}"
-		>{contributor.name}</a>
+			style="background-position: {100 * i / (contributors.length - 1)}% 0"
+			href="https://github.com/{contributor}"
```

Comment:
I think here and below should be `{contributor.name}`. Because links not working now.

Reply:
Hm strange, it works locally but not in prod. Maybe cached version of `_contributors.js`?

Reply:
I think it might just have been a broken deploy — it's supposed to regenerate that file but somehow it failed, maybe? Have redeployed and it looks ok for me